assets/css/src/modal.styl

```diff
@@ -28,24 +31,21 @@ body.mailpoet_modal_opened
   box-shadow(0px 0px 20px 2px alpha(#fff, 75%))
 
 // overlay: state
-#mailpoet_modal_overlay.mailpoet_overlay_hidden
+.mailpoet_modal_overlay.mailpoet_overlay_hidden
   background-color: transparent
 
 // overlay: state
-#mailpoet_modal_overlay.mailpoet_overlay_loading
+.mailpoet_modal_overlay.mailpoet_overlay_loading
   background-color: overlay_background_color !important
-  display: block !important
+  display: flex !important
```

Comment:
Can we get rid of those pesky `!important` statements?

Reply:
Not this one, without significantly refactoring modal.js. It's controlled with jQuery's `.show()` and `.hide()`, which uses inline styles and only `!important` can beat that from CSS file.